Mono.Debugging/Mono.Debugging.Client/StackFrame.cs

```diff
@@ -135,7 +135,16 @@ internal void Attach (DebuggerSession debugSession)
 		/// </summary>
 		public string GetLocationSignature ()
 		{
-			return $"{Path.GetFileNameWithoutExtension (this.FullModuleName)}!{this.SourceLocation.MethodName}:{this.Address}";
+			string methodName = this.SourceLocation.MethodName;
+			if (methodName != null && !methodName.Contains("!")) {
```

Comment:
This optimizes to a SIMD contains:

```suggestion
			if (methodName != null && !methodName.Contains('!')) {
```

Reply:
This is probably as far from a perf bottleneck as it gets, would really be a microoptimization.

Also .NET Framework (and Mono) don’t have a string.Contains(char), and I’d prefer to not call via an interface.